docs/tutorials/tfx/airflow_workshop.md

```diff
@@ -276,7 +276,7 @@ airflow webserver -p 7070
 
 #### Airflow CLI
 
-You can also use the [Airflow CLI](https://airflow.apache.org/cli.html) to
+You can also use the [Airflow CLI]([https://airflow.apache.org/cli.html](https://airflow.apache.org/docs/apache-airflow/stable/cli-and-env-variables-ref.html) to
```

Comment:
It seems like the Markdown syntax is a bit mixed. Could you make sure that this is the right change?